app/assets/javascripts/custom_play_button.js

```diff
@@ -1,41 +1,59 @@
 $(document).ready(function() {
-  if (Modernizr.touch && !Foundation.MediaQuery.atLeast('large')) {
-    $('.play-button').hide();
-    $('video').each(function() {
-      $(this).prop('controls', true);
-      $(this).prop('controlsList', 'nodownload');
-    });
-  }
+  adjustVideoControlsToScreenSize();
+
+  $(window).resize(function() {
+    adjustVideoControlsToScreenSize();
+  });
 
-  $('video').click(function(e) {
-    if ($('.play-button').length) {
+  $("video").click(function(e) {
+    if ($(".play-button").length) {
       e.preventDefault();
       videoResponse(this);
     }
   });
 
-  $('.play-button').click(function(e) {
+  $(".play-button").click(function(e) {
     e.preventDefault();
     videoResponse(this.nextElementSibling);
   });
 
+  function adjustVideoControlsToScreenSize() {
+    if (Modernizr.touch && !Foundation.MediaQuery.atLeast("large")) {
```

Comment:
There needs to be an else to this situation where you show the play button and hide the controls.

Reply:
I'm not sure I understand why this is necessary - could you elaborate? :)

Reply:
As in -- isn't the else case you're describing the default situation? So if the 'if' conditions aren't met it just defaults back to that anyway?